libs-ts/cover-component/src/helpers.ts

```diff
@@ -78,15 +77,6 @@ function ensureRoman(str: string): string {
   return str.match(/^\d+$/) ? toRoman(Number(str)) : str;
 }
 
-export function formatBlurb(blurb: string): string {
```

Comment:
this allowed removing `adoc-utils`. and this function was not doing anything, for a while we have pre-quoted everything in the database, this was a leftover thingy from years ago that should have been deleted anyway, and gives us big bundle/perf improvements.